sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (Unreleased)
 
-- Using `Poller` and `PollerLike` from the latest `@azure/core-lro` instead of `KVPoller` and `KVPollerLike`.
+- Added the optional `serviceVersion` property to the `CertificateClient` optional parameters.
+  It defaults to the latest supported API version, which currently is `7.1`.
+- Added `recoverableDays` as an optional property to `KeyProperties`.
+- Fixed a bug on `importCertificate`. The bytes of a PEM formatted certificate were being encoded again on base64.
+  Now, if the user provides a policy with contentType `application/x-pem-file`, the bytes will be assumed to be formatted using ASCII,
+  and thus will be sent as is.
+- Now using `Poller` and `PollerLike` from the latest `@azure/core-lro` instead of `KVPoller` and `KVPollerLike`.
 - `KVPollerLike` is now an alias of `PollerLike`.
 - `KVPollerLike` is considered deprecated. Use `PollerLike`.
-- Fixed [bug 8378](https://github.com/Azure/azure-sdk-for-js/issues/8378), which caused the challenge based authentication to re-authenticate on every new request.
+
```

Comment:
https://gist.github.com/sadasant/2cb528aa8502e1b14fd1552516bacafb/revisions shows a `CertificateKeyType` supporting a new type `oct`. Should we talk about that here?

Reply:
Yes! Thank you.

Reply:
I've updated this, please let me know how it looks!